clippy_lints/src/enum_variants.rs

```diff
@@ -212,6 +212,11 @@ fn check_variant(cx: &LateContext<'_>, threshold: u64, def: &EnumDef<'_>, item_n
     );
 }
 
+#[must_use]
+fn have_no_extra_prefix(prefixes: &Vec<&str>) -> bool {
+    prefixes.is_empty() || prefixes.join("") == "_"
```

Comment:
```suggestion
    prefixes.all(|p| p == "" || p == "_")
```
would admittedly allow multiple underscores (is that a bad thing?), but avoid the overhead of `join`.

Reply:
I fixed in https://github.com/rust-lang/rust-clippy/pull/9032/commits/d827b834d91383620d587fb4a1e74d8579e3a280